packages/@uppy/google-photos/src/GooglePhotosProviderViews.ts

```diff
@@ -0,0 +1,7 @@
+import { ProviderViews } from '@uppy/provider-views'
+import type { Body, Meta } from '@uppy/utils/lib/UppyFile'
+
+export default class GooglePhotosProviderViews<
```

Comment:
Why do we need this?

Reply:
not sure, I just did the same as for google drive, for consistency: https://github.com/transloadit/uppy/blame/main/packages/%40uppy/google-drive/src/DriveProviderViews.ts

Reply:
Let's remove it then